website/content/docs/user-guide/cloudflare-setup.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Cloudflare Setup"
+weight: 3
+---
+
+To use Emma with Cloudflare, you need to set up the following services:
```

Comment:
the important piece is that the `emma init` command takes care of all those steps